tools/diff/diff.xml

```diff
@@ -0,0 +1,91 @@
+<tool id="diff" name="diff" version="@TOOL_VERSION@+@WRAPPER_VERSION@">
+
+    <description>analyzes two files and generates a text file with information about the lines that are different.</description>
```

Comment:
Do you know if `diff` is part of the coreutils package? If so we should probably depend on this package.

Reply:
Added conda-forge package for diffutils 3.6 and requirement 😃